static/app/views/dashboards/editAccessSelector.tsx

```diff
@@ -216,7 +217,15 @@ function EditAccessSelector({dashboard, onChangeEditAccess}: EditAccessSelectorP
       searchable
       options={allDropdownOptions}
       value={selectedOptions}
-      triggerLabel={[t('Edit Access:'), triggerAvatars]}
+      triggerLabel={[
+        t('Edit Access:'),
+        triggerAvatars,
+        <FeatureBadge
+          key="beta-badge"
+          type="beta"
+          title={'This feature is available for early adopters and may change'}
```

Comment:
two things:
1. it'll be nice to wrap this in a t() because it's user facing
2. instead of saying its available for EA, i've seen that most people use "This feature is currently in beta. Beta features may change etc....". Unless this text has been given to you to use by someone else then ignore me 😄